conmon/conmon.c

```diff
@@ -398,6 +405,11 @@ static void reopen_log_file(void)
 {
 	_cleanup_free_ char *opt_log_path_tmp = g_strdup_printf("%s.tmp", opt_log_path);
 
+	/* Sync the logs to disk */
+	if (fsync(log_fd) < 0) {
```

Comment:
we are going to create a new file few lines below, so this should not make a difference (unless the file cannot be re-opened), either way it doesn't hurt